lib/Runtime/Debug/DiagObjectModel.h

```diff
@@ -149,6 +149,7 @@ namespace Js
         FW_EnumWithScopeAlso    = 0x2,  // While walking include the with scope as well.
         FW_AllowLexicalThis     = 0x4,  // Do not filter out Js::PropertyIds::_lexicalThisSlotSymbol
         FW_AllowSuperReference  = 0x8,  // Allow walking of Js::PropertyIds::_superReferenceSymbol and Js::PropertyIds::_superCtorReferenceSymbol
+        FW_SkipGlobal           = 0x10, // Do not add global object directly.
```

Comment:
I think more appropriate name is FW_DontAddGlobalsDirectly as this is not skipping globals as the name suggest without reading the comment.